reporter/src/funTest/assets/static-html-reporter-test-input.yml

```diff
@@ -340,16 +340,22 @@ scanner:
 evaluator:
   start_time: "1970-01-01T00:00:00Z"
   end_time: "1970-01-01T00:00:00Z"
-  errors:
-  - timestamp: "2019-01-02T17:23:55.486Z"
-    source: "GPL-1.0-only"
-    message: "GPL 1 error"
+  violations:
+  - rule: "2019-01-02T17:23:55.486Z"
```

Comment:
Why is the rule just a timestamp? Shouldn't it be a name or something?

Reply:
Fixed.